### ISSUE TITLE
about loss and train_accuracy

### ISSUE THREAD

**Issue by myh12138** [tommyMessi/PST-table#13]
why the loss use log_pointer_score not argmax_pointer,but train_accuracy use argmax_pointer
unrolled = log_pointer_score.view(-1, log_pointer_score.size(-1))
loss = F.nll_loss(unrolled, target.view(-1), ignore_index=-1)
